dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/aspect/Counted.java

```diff
@@ -0,0 +1,11 @@
+package uk.gov.justice.laa.crime.dces.integration.aspect;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Counted {
+}
```

Comment:
Probably dumb question before approving: why this instead of https://github.com/micrometer-metrics/micrometer/blob/main/micrometer-core/src/main/java/io/micrometer/core/annotation/Counted.java annotation (and aspect) built in to Micrometer / Spring Boot?

https://docs.spring.io/spring-boot/reference/actuator/observability.html#actuator.observability.annotations

Reply:
OK - sounds like something we should come back to and check again in future, in case a future version of Spring Boot / micrometer adds that flexibility, but sounds good.